app/src/main/java/net/frju/flym/data/entities/Entry.kt

```diff
@@ -68,7 +68,11 @@ fun SyndEntry.toDbFormat(feed: Feed): Entry {
 	?: UUID.randomUUID().toString())).sha1()
 	item.feedId = feed.id
     @Suppress("DEPRECATION")
-    item.title = Html.fromHtml(title).toString()
+	if (title != null) {
+		item.title = Html.fromHtml(title).toString()
+	} else {
+		item.title = "Untitled"
```

Comment:
it should be put inside the english strings.xml file (no need for you to add the other languages)

Reply:
Could you guide: how do I get strings from the xml from within Entry.kt? It would be trivial to do in Activity, but Entry.kt does not have access to `Context` (that I am aware of)

Reply:
Sure thing!